### PR TITLE
Added requireArray validator

### PR DESCRIPTION
**Pull request by mattsmee**

### CODE REVIEW

File: lib/check.js [hunk 1]
```diff
@@ -12,6 +12,7 @@ const FAIL = (msg) => _.always(msg);
 
 const MSG  = {
   REQUIRED           : 'required'
+, REQUIRE_ARRAY     : 'require_array'
```
Comment (jonajonlee) [inline lib/check.js]: fix this right now. i freaking hate it

Comment (tiggaplease) [inline lib/check.js]: OCD....

**Comment by jonajonlee** [comment]
:shipit:

**Comment by tiggaplease** [comment]
:shipit:

**Pull request merged**